components/activity_and_insights/insights/card/card.scss

```diff
@@ -0,0 +1,75 @@
+.Card.insights-card {
+    border: 1px solid rgba(var(--center-channel-color-rgb), 0.08);
+    margin: 8px;
+    border-radius: 8px;
+    box-shadow: none;
+
+    &.large {
+        width: auto;
+    }
+
+    &.medium {
+        width: calc((100%/2) - 16px);
+        float: left;
+    }
+
+    &.small {
+        width: calc((100%/3) - 16px);
+        float: left;
+    }
+
+    .Card__header {
+        display: flex;
+        align-items: center;
+        padding: 24px 24px 12px;
+
+        &:hover {
+            cursor: pointer;
+        }
+
+        .title-and-subtitle {
+            display: flex;
+            flex-direction: column;
+
+            .text-top {
+                display: block;
+
+                h2 {
+                    margin: 0;
+                    font-size: 16px;
+                    font-weight: 600;
+                    line-height: 24px;
+                }
+            }
+
+            .text-bottom {
+                display: block;
+                color: rgba(var(--center-channel-color-rgb), 0.72);
+                font-size: 12px;
+                font-weight: 400;
+                line-height: 16px;
+            }
+        }
+
+        span.icon {
```

Comment:
Recommend we position this absolutely instead and do some other stylistic changes:

We should also change this to a `button` rather than a `span` tag

```
position: absolute;
top: 20px;
right: 20px;
padding: 8px;
border-radius: 4px;
color: rgba(var(--center-channel-color-rgb),.56);
```

We should also add a hover state to this as a button and make these style changes on hover:

```
color: rgba(var(--center-channel-color-rgb),.56);
background-color: rgba(var(--center-channel-color-rgb),.08);
```

Reply:
@matthewbirtch would you want the whole header clickable for opening the modal or just the button?

Reply:
Suggested changes are done

Reply:
> @matthewbirtch would you want the whole header clickable for opening the modal or just the button? 

I'm good with the whole header and the button both being clickable